init_app/init_env_vars.go

```diff
@@ -0,0 +1,16 @@
+package init_app
+
+import (
+	"os"
+)
+
+func InitEnvVars() string {
+	ginMode := "debug"
```

Comment:
Just checking you still want to run this in debug?

Reply:
This bit of the code is a bit misleading (my bad it's adopted from label-pods) but basically it defaults to debug if there's now value set otherwise it will use the value set in the env var